packages/android-app/android/xnative/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="uk.co.thetimes.xnative">
+
+  <application
```

Comment:
I'd remove this since we aren't publishing an app here, and we don't want to end up in our app manifest

Reply:
51bbdd0